env.go

```diff
@@ -43,14 +44,16 @@ func suffix(s string, suffix byte) string {
 	}
 }
 
-func (p *Env) SetEnvDir(envDir string) error {
-	return eachEnvFile(envDir, func(k, v string) error {
-		return p.Setenv(k, v)
-	})
-}
-
 func (p *Env) AddEnvDir(envDir string) error {
 	return eachEnvFile(envDir, func(k, v string) error {
+		if strings.HasSuffix(k, ".append") {
+			name := strings.TrimSuffix(k, ".append")
+			return p.Setenv(name, p.Getenv(name)+string(v))
```

Comment:
Will `Setenv` expand any `$ENV_VARS` in `v`? That is, can you put env vars in the value of an env var? I don't have a preference here, but wonder if we should be explicit about that behavior in the spec.

Reply:
`Setenv` is `os.Setenv`, which does not to expand. The spec should definitely be explicit about this.

Reply:
Clarified: https://github.com/buildpack/spec/commit/032f1be2e39235b5b471b05b8004e4b6b12d6763